types/amcharts/amcharts-tests.ts

```diff
@@ -0,0 +1,7 @@
+import { AmCharts } from "amcharts";
+
+// TODO: provide much more tests
```

Comment:
Adding a bunch more tests would help a bunch. Easiest would be to copy/paste the samples from the documentation to make sure at least those work.

Reply:
Thank you, I'll come back with a bunch of new tests within next few days.